### DISCUSSION TITLE
Less than and greater than symbols < > <>

### DISCUSSION THREAD

**Discussion by jonch070** [manna-harbour/miryoku#275] [Q&A] [answered]
Hello! trying to learn a qwerty miryoku with CAGS homerow mods. https://configure.zsa.io/ergodox-ez/layouts/Zebwv/nVQgb/0

As a total beginner, where are < and > symbols? how should I best deal with these?

**Comment by KAGEYAM4** [comment, marked as answer]
@jonch070  <,> are on base layer  on `,` and `.` like normal keyboard and you will have to use shift to access them.

Read the reference manual from here -  https://github.com/manna-harbour/miryoku/tree/master/docs/reference
